SEM_2_PRACTICALS/prac_06/guitar_test.py

```diff
@@ -0,0 +1,16 @@
+from SEM_2_PRACTICALS.prac_06.guitar import Guitar
+
+
+def main():
+    gibson = Guitar("Gibson L-5 CES", 1922, 16035.4)
+    another_guitar = Guitar("Another Guitar", 2013)
+    age = gibson.get_age()
```

Comment:
These temporary variables are inconsistent (you don't use them later) and could lead to errors.